homework-submission/homework-week1/homework-week2/App.js

```diff
@@ -0,0 +1,143 @@
+const ListItem = ({ id, description, deadline, clickHandler, className }) => {
+  return (
+    <li key={id} id={id} onClick={clickHandler} className={className}>
+      {description} , {deadline}
+    </li>
+  );
+};
+
+const Button = ({ type, text, className }) => {
+  return (
+    <button type={type} className={className}>
+      {text}
+    </button>
+  );
+};
+
+const DynamicList = ({ List, clickHandler, className }) => {
```

Comment:
As list is not a component it should be with a small letter l(list).

Reply:
I have done.